src/main/java/org/alfresco/repo/rendition2/TransformationOptionsConverter.java

```diff
@@ -259,12 +264,19 @@ else if (IMAGE_OPTIONS.containsAll(subclassOptionNames) ||  PDF_OPTIONS.contains
                 }
             }
         }
+        else if (!hasOptions)
```

Comment:
If isPdfRendition is true then it will not execute this code, is that the intended behaviour? The comment states that it should be used by "pdf" rendition...

Reply:
Thanks, you are correct that there is something wrong here. I think the 'if check' is redundant. 

Re: comment about pdf, also see comment on line 185